raftify/src/raft_node/mod.rs

```diff
@@ -16,6 +16,7 @@ use std::{
     },
     time::{Duration, Instant},
 };
+use jopemachine_raft::eraftpb::MessageType;
```

Comment:
Please update the import statements to follow the style shown below.

```suggestion
use crate::raft::eraftpb::MessageType;
```

The package name `jopemachine_raft` is not appropriate and is only temporary.